src/main/java/study/racing/domain/CarNames.java

```diff
@@ -0,0 +1,16 @@
+package study.racing.domain;
+
+import java.util.List;
+
+public class CarNames {
+
+    private final List<CarName> names;
+
```

Comment:
```suggestion
    public CarNames(String names) {
        this(names.split(","));
    }

    public CarNames(String[] names) {
        this(Arrays.stream(names)
                .map(Name::new)
                .toList());
    }
```

CarNames 분리 시도 👍 
이 객체에도 부 생성자를 추가해 보면 CarNames를 사용하는 개발자 입장에서 편리하지 않을까?

Reply:
말씀하신 것처럼 개발자 입장에서 훨씬 편리하다고 느껴집니다 !
앞으로 클래스 생성할 때 부 생성자 관련해서 살펴보는 습관을 가져보겠습니다☺️